app/views/schedule/scheduled_publishing_datetime.html.erb

```diff
@@ -0,0 +1,89 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("schedule.scheduled_publishing_datetime.title") %>
+
+<% current_scheduled_publish_datetime = @edition.revision.scheduled_publishing_datetime %>
+<% default_date = Time.zone.now.tomorrow %>
+<% publish_date = current_scheduled_publish_datetime || default_date %>
+<% schedule_param = params["schedule"] ? params["schedule"] : {} %>
+<% selected_time = schedule_param["time"] || scheduled_time(current_scheduled_publish_datetime) %>
+<% action_not_selected = @issues&.select { |i| i.issue_key == :action_not_selected }&.first&.to_item %>
```

Comment:
Could we use `@issues&.items_for` here, to avoid the manual select?

Reply:
That's something that is resolved in my PR